system/Router/RouteCollection.php

```diff
@@ -988,8 +989,13 @@ public function presenter(string $name, array $options = null): RouteCollectionI
 	 *
 	 * @return RouteCollectionInterface
 	 */
-	public function match(array $verbs = [], string $from, $to, array $options = null): RouteCollectionInterface
+	public function match(array $verbs = [], string $from = '', $to = '', array $options = null): RouteCollectionInterface
 	{
+		if (empty($from) || empty($to))
```

Comment:
use empty string comparison as well?

Reply:
These are all good suggestions, but belonged in the original PR that was cherry-picked. I don't have time to wait for tests again but you (or anyone) could apply these in a new PR after this. Please review for the content at hand.